### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by stackotter** [stackotter/delta-client#13]
Due to the fact that the client uses SwiftUI, almost the entire client could be ported to iOS pretty easily. There are just a few things such as file storage, input (touch screen), and the way the renderer is integrated into SwiftUI that would need iOS specific implementations. `swift-bundler` would also need to have iOS build support added. If you wanna try implementing this one you can hop on the Discord server and we can chat about how best to integrate iOS builds into `swift-bundler`. The best way would probably be building using swiftpm and then creating an ipa (iOS app file) just like for macOS apps (as opposed to delegating the ipa creation to Xcode).

**Comment by ninjadev64** [comment]
Shouldn't this also mention tvOS?

**Comment by stackotter** [comment]
Nah, that's a different thing. That's quite a bit further down the line probably so I didn't make an issue for it.

**Comment by ninjadev64** [comment]
> Nah, that's a different thing. That's quite a bit further down the line probably so I didn't make an issue for it.

Oh OK, I've seen older Apple TVs showing that they run older iOS versions, so I thought tvOS on newer models must be similar to/a fork of iOS.

**Comment by stackotter** [comment]
The underlying system is pretty similar, and the app format is probably similar. But the interface and controls are quite different and would take quite a bit of work (enough work to be a separate issue at least).

**Comment by stackotter** [comment]
This should be possible soon, I'm working on iOS support for Swift Bundler

**Comment by stackotter** [comment]
The `ios` branch now has basic iOS support. I haven't made any iOS specific changes to the UI, so it isn't very functional, but I have now successfully joined a world on my iPhone 7. Compiling and running on iOS requires the sponsor-only version of Swift Bundler.
